src/components/Svg/index.tsx

```diff
@@ -12,6 +12,7 @@ export { default as CommunityIcon } from "./Icons/Community";
 export { default as ErrorIcon } from "./Icons/Error";
```

Comment:
Is there a reason why you don't export other icons ?

Reply:
Just missed it.